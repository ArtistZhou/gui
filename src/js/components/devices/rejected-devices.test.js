import React from 'react';
import renderer from 'react-test-renderer';
import Rejected from './rejected-devices';

it('renders correctly', () => {
  const tree = renderer.create(<Rejected />).toJSON();
  expect(tree).toMatchSnapshot();
});
