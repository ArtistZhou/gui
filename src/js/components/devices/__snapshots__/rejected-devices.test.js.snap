// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="tab-container"
>
  <div
    className="loaderContainer"
  >
    <div
      className="loader"
    >
      <span
        className="dot dot_1"
      />
      <span
        className="dot dot_2"
      />
      <span
        className="dot dot_3"
      />
      <span
        className="dot dot_4"
      />
    </div>
  </div>
  <div
    className="hidden"
  >
    <p>
      There are no rejected devices
    </p>
  </div>
</div>
`;
